dashboard/db/migrate/20240307095343_create_lti_feedbacks.rb

```diff
@@ -0,0 +1,13 @@
+class CreateLtiFeedbacks < ActiveRecord::Migration[6.1]
+  def change
+    create_table :lti_feedbacks do |t|
```

Comment:
Plural of "feedback" is still "feedback" I believe

Reply:
That's what Rails generated with `rails g model Lti::Feedback`. Do you think we should use `lti_feedback` or maybe something like `lti_feedback_items` for the table name?

Reply:
Yeah I figured it was the automatic pluralization not knowing about "feedback." IMO just `lti_feedback` is fine.

Reply:
I've noticed that we are already using the plural form of this word. Therefore, if we aim to apply the plural form fix correctly by adding it to the Rails [inflector](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/initializers/inflections.rb#L7), we must also change the names of the remaining tables accordingly. Otherwise, we might break some ActiveRecord functionality for those models.
https://github.com/code-dot-org/code-dot-org/blob/fa9ebf57df02525072d11688f6cbfb4c008cf15a/dashboard/db/schema.rb#L2013
https://github.com/code-dot-org/code-dot-org/blob/fa9ebf57df02525072d11688f6cbfb4c008cf15a/dashboard/db/schema.rb#L614

Reply:
Ok - don't think this is worth the risk of making the change to the inflector. Let's just leave it as is.